docs/configuration/config-modification.md

```diff
@@ -82,19 +47,19 @@ See exemplary component configuration in TypeScript and JavaScript.
 ```tsx
 import * as React from "react";
 import { render } from "react-dom";
-import AsyncApiComponent, { ConfigInterface } from "asyncapi-react";
+import AsyncApiUI, { ConfigInterface } from "@asyncapi/react-component";
```

Comment:
why AsyncAPIUI? it is AsyncAPIComponent after all

Reply:
Maybe we rename it to `@asyncapi/ui` -> https://github.com/asyncapi/asyncapi-react/issues/263, but yeah, I will leave it as `AsyncApiComponent`.